lib/events.js

```diff
@@ -525,3 +525,17 @@ function unwrapListeners(arr) {
   }
   return ret;
 }
+
+EventEmitter.prototype.onMultiple = function onMultiple(obj) {
+
+    for(let event in obj) {
+
+      const functionArray = obj[event];
+
+      for(let i = 0; i < functionArray.length; i++) {
```

Comment:
This should allow for the simple case also:

```js
ee.onMultiple({
  event1() { },
  event2() { }
});
```

Reply:
Not sure what that object represents. Is it like an array?